stackrox/base/rbac.authorization.k8s.io/rolebindings/stackrox-central-diagnostics.yaml

```diff
@@ -0,0 +1,29 @@
+---
+# Source: stackrox-central-services/templates/01-central-03-diagnostics-rbac.yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
move the folders `roles` and `rolebindings` into `base` and get rid of `rbac.authorization.k8s.io` so that it is consistent with the rest of the structure